src/box/bind.c

```diff
@@ -175,12 +175,8 @@ sql_bind_list_decode(const char *data, struct sql_bind **out_bind)
 	struct region *region = &fiber()->gc;
 	uint32_t used = region_used(region);
 	size_t size;
-	struct sql_bind *bind = region_alloc_array(region, typeof(bind[0]),
-						   bind_count, &size);
-	if (bind == NULL) {
-		diag_set(OutOfMemory, size, "region_alloc_array", "bind");
-		return -1;
-	}
+	struct sql_bind *bind = xregion_alloc_array(region, typeof(bind[0]),
+						    bind_count, &size);
```

Comment:
Would you mind doing this simple one first? https://github.com/tarantool/tarantool/issues/8522. Otherwise all those places would have to be updated again after that ticket.

Reply:
Thank you for the suggestion! Done